integrations/issue_test.go

```diff
@@ -184,3 +187,34 @@ func TestIssueCommentClose(t *testing.T) {
 	val := htmlDoc.doc.Find(".comment-list .comments .comment .render-content p").First().Text()
 	assert.Equal(t, "Description", val)
 }
+
+func TestNewIssueConcurrent(t *testing.T) {
+
+	prepareTestEnv(t)
+	session := loginUser(t, "user2")
+
+	f := func(ctx context.Context, cancel context.CancelFunc, wg *sync.WaitGroup) {
+		defer wg.Done()
+
+		select {
+		case <-ctx.Done():
+			return
+		default:
+		}
+		testNewIssue(t, session, "user2", "repo1", fmt.Sprintf("Title"), "Description")
```

Comment:
You are stressing the database here (what you need) but also stressing other layers, like the http client & server, the computer resources, etc. I think it would be preferrable if you'd call the models/issue.go methods instead.

Reply:
Agreed. I'll try to figure out how to do that.